applications: nrf_desktop: Describle BLE peer control on DK (#22304)
nrfconnect/sdk-nrf: applications/nrf_desktop/description.rst

            * Middle position: Select the Bluetooth LE peers.
            * Bottom position: Mouse turned off.

          By default, when the dongle peer is selected, the peer control is disabled until the switch is set to another position.

>           The dongle peer erase can be enabled in configuration.

Reviewer: ```suggestion
          You can enable the dongle peer erase in the configuration.
```

          After the forced scan timeout, the scan is interrupted if another peripheral connected to the dongle is active.

          .. note::
             |led_note|

        * |nRF_Desktop_cancel_operation|

   .. tab:: nRF54 DK

>       The following predefined button is assigned to peer control operations for an nRF54H DK.

Reviewer: ```suggestion
      The following predefined button is assigned to peer control operations for an nRF54H Series DK.
```

>       The following predefined button is assigned to peer control operations for an nRF54H Series DK.

Reviewer 2: What about 54L?

Author: Nice catch. Thanks!

Author: Updated on whole page

>       Button 0
>          * If the nRF54 DK acts as a dongle:
> 
>             * Long-press the **Button 0** to initialize peer erase.
>               When **LED1** starts blinking rapidly, double-press to confirm the operation.
>               After the confirmation, all the Bluetooth bonds are removed for the dongle.
>             * Short-press to start scanning for both bonded and non-bonded Bluetooth Peripherals.
>               After the forced scan timeout, the scan is interrupted if another peripheral connected to the dongle is active.
> 
>               .. note::
>                  |led_note|
> 
>             * |nRF_Desktop_cancel_operation|
> 
>          * If the nRF54 DK acts as a peripheral:
> 
>             * Press the **Button 0** before the nRF54 DK is powered up with the on/off switch.
>               Long-press to initialize and confirm the peer erase.
>               |nRF_Desktop_confirmation_effect|
> 
>               .. note::
>                  |led_note|
> 
>             * |nRF_Desktop_cancel_operation|

Reviewer 3: Let's make this part of the content the same as for the other tab. The only difference should be the Button and LED labels.

```suggestion
      Button 0
         * If the DK acts as a dongle:

            * Long-press the **Button 0** to initialize peer erase.
              When **LED1** starts blinking rapidly, double-press to confirm the operation.
              After the confirmation, all the Bluetooth bonds are removed for the dongle.
            * Short-press to start scanning for both bonded and non-bonded Bluetooth Peripherals.
              After the forced scan timeout, the scan is interrupted if another peripheral connected to the dongle is active.

              .. note::
                 |led_note|

            * |nRF_Desktop_cancel_operation|

         * If the DK acts as a peripheral:

            * Press the **Button 0** before the DK is powered up with the on/off switch.
              Long-press to initialize and confirm the peer erase.
              |nRF_Desktop_confirmation_effect|

              .. note::
                 |led_note|

            * |nRF_Desktop_cancel_operation|
```

   .. tab:: nRF52 and nRF53 DKs

>       The following predefined button is assigned to peer control operations for an nRF52 or nRF53 DK.

Reviewer: ```suggestion
      The following predefined button is assigned to peer control operations for an nRF52 or nRF53 Series DK.
```

>       Button 0

Reviewer 3: ```suggestion
      Button 1
```

      **LED1** is used for the system state indication.
      It is located in the bottom right corner of the dongle, next to the USB connector.

>    .. tab:: nRF54H DK

Reviewer 3: The LED indiication tab uses the "nRF54H DK" category while the peer control button uses "nRF54 DK"

Author: Thanks. Updated on whole page

=======

You can build and test the application in various configurations.

.. note::
   The following procedure refers to the scenario where the gaming mouse (nRF52840 Gaming Mouse) and the keyboard (nRF52832 Desktop Keyboard) are connected simultaneously to the dongle (nRF52840 USB Dongle).

>    You could perform similar tests with nRF52, nRF53 or nRF54 Development Kits.

Reviewer: ```suggestion
   You can perform similar tests with nRF52, nRF53, or nRF54 Series DKs.
```
